zipline/testing/fixtures.py

```diff
@@ -1192,10 +1218,21 @@ def init_class_fixtures(cls):
 
         cls.hdf5_daily_bar_path = p = cls.make_hdf5_daily_bar_path()
 
-        f = HDF5DailyBarWriter(p, date_chunk_size=30).write_from_sid_df_pairs(
-            cls.HDF5_DAILY_BAR_COUNTRY_CODE,
-            cls.make_equity_daily_bar_data(),
-        )
+        writer = HDF5DailyBarWriter(p, date_chunk_size=30)
+
+        for country_code in cls.HDF5_DAILY_BAR_COUNTRY_CODES:
+            sids = [
+                sid for sid in cls.asset_finder.equities_sids
+                if (
+                    cls.asset_finder.retrieve_asset(sid).country_code
```

Comment:
What do you think about moving this method into the asset finder as something like `equity_sids_for_country_code`. Even if we keep the current impl for now, in the future we can probably make this faster by going straight to the sqlite db.

Reply:
(we actually might be able to just piggyback on the country-parametric lifetimes machinery for this).